### ISSUE TITLE
The Pre-game/Pre-season

### ISSUE THREAD

**Issue by mikekasprzak** [JammerCore/JammerCore#1205]
Ludum Dare introduces many people to making games, but can do a way better job of it.

I'm going to refer to this concept as the **"pre-game"**, i.e. what we do before an event.

The pre-game right now works like this.

![pregame](https://user-images.githubusercontent.com/1570313/29846667-d30554e0-8ce5-11e7-8da5-05a52c5f54f6.png)

It's a kind-of ugly blob that tries to convince you we're cool. It doesn't do a very good job of it. :wink:

What it should be doing instead is highlighting _"hey, you can make a game"_. From that you can go 1 of 2 places:

* Learn how make a game
* Learn about some of the great games that came from Ludum Dare

Currently we have a poor implementation of the 2nd one, sending folks to our steam curator. We should be showcasing these much more boldly. i.e. silent mini trailers, and whatnot.

Exploring is only half the battle though. Inspiring people to make a game, that is our focus. That is why we're still here after 15 years.

twitch.tv kind-of does this now.

They have the non-user view (not logged in).

![homey](https://user-images.githubusercontent.com/1570313/29846812-768a1204-8ce6-11e7-9942-e94235322e29.png)

Another view when you are logged in (combined feed and streams).

![home2](https://user-images.githubusercontent.com/1570313/29846825-864e2824-8ce6-11e7-92dd-380c18b96473.png)

I'm not entirely happy with their layout, but I do think the concept is strong.

Also new with the new website is the concept of signing up for an event. This needs to play a bigger role, and signups should open sooner. Once you've signed up, you're in a different mode. You're getting ready to participate, so we should be showing you things to motivate you. Articles on how to start making games with tools, success stories, etc.

Also real world events need to be highlighted better (i.e. at all). Though not required to participate, it's an exciting part of things.

Countdown clock should always be going, but it can be more subtle.

There's also the Ludum Dare event education side. Not only should be we educate you how to make games, we should educate you about participation. **You should go in to Ludum Dare expecting to play and rate other people's games**.

Bottom line, we need to introduce you to Ludum Dare (and game making) better.

**Comment by local-minimum** [comment]
One way to ease in doing this better, I think is getting more tags on the platform. If there were 'tutorial' and 'post-mortem' and some more like this. I would make special feeds easier to construct.

**Comment by povrazor** [comment]
Actually that reminds me that tags could be used for more than just ... well, typically tagging. There could be restricted tags that normal users can't add, such as featured games, or things that require approval. Right now tags are a free-for-all, only really limited by what tags are on the platform list.
